packages/input-option/src/InputOption/InputOption.story.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
```

Comment:
Process question -- as we release new components, how are we ensuring that this is compatible with .design?

Reply:
We import directly from Storybook so I feel like it should mostly be okay, but we don't have anything set up to test with the website overall. We'd need some draft state of these components to be consumed from our staging environment if we want that.

Alpha release from a staging branch that's consumed in our staging environment maybe?

Reply:
Some sort of pre-release, or `yarn link`ing this repo to the design repo.

I agree though, we do need some explicit guidelines on how to prep a story file for use on the website